### DISCUSSION TITLE
Questions - may I include UABEA in a publicly distributed patch file?

### DISCUSSION THREAD

**Discussion by enarkay** [nesrak1/UABEA#256] [Q&A] [answered]
I make fan localization patches from time to time, but as you may know, maintaining a localization patch for a game that hasn't stopped updating can be quite an annoyance. Not only can the patch files get big depending on how many assets I need to replace and overwrite, but a game update can break the whole patch even if there were no changes in terms of localization materials. In such case, I'd have to update and maintain the localization patch every time the game updates.

So while I haven't tried out the feature yet, I was thinking of trying out the mod maker (package file) feature of UABEA. But in such case, to make things as easy as possible for users to use/install the localization patch, I was thinking it would be nice if I could include all of UABEA, the package file, and a BAT file to execute the command to have UABEA install the package file for the users.

However, before I tried this method out, I wanted to know for sure if it's possible and okay with you for me to include UABEA in a publicly distributed localization patch, for the above mentioned purpose.

**Comment by nesrak1** [comment, marked as answer]
The project is MIT licensed. You can read the LICENSE page for more info: https://github.com/nesrak1/UABEA/blob/master/license. In short: yes, you can redistribute UABEA following the license requirements.

**Reply by enarkay** [reply]
Thank you for your answer as well as telling me where I should've looked first. All this time I've never thought to look for a license in github projects. Now that I know, I will be able to look for licenses before I start asking similar questions. Thanks! :)
